.readthedocs.yaml

```diff
@@ -11,13 +11,17 @@ build:
   os: "ubuntu-20.04"
   tools:
     python: "3.9"
+  commands:
+    - pip install poetry
```

Comment:
If this lets us ditch `requirements.txt` in factor of something like `poetry install -E docs`... 🔥